lib/services/elasticsearch.js

```diff
@@ -38,6 +38,60 @@ function ElasticSearch (kuzzle, options, config) {
     }
   });
 
+  this.errorMessagesMapping = [
+    {
+      // [illegal_argument_exception] object mapping [titi] can't be changed from nested to non-nested
+      regex: /^\[illegal_argument_exception\] object mapping \[(.*?)\] can't be changed from nested to non-nested$/,
+      replacement: 'Can not change mapping for field "$1" from nested to another type'
```

Comment:
hmmm 
for me, "nested" is not a "type", but rather a property of an object.
an object can be either nested or non-nested.
a scalar can only be non-nested

So "from nested to another type" doesn't make sense